Add AggregatorRouter contract explanation with example code (#48)
Bisonai/orakl-docs: en/developers-guide/data-feed.md

address currentAggregator = dataFeed.aggregator()
```

### Use Aggregator Router

> - Conviniently implement datafeed by using router contracts

Reviewer: ```suggestion
- Conveniently access data feeds using `AggregatorRouter` contract
```

> - Access all functions in aggregator by including price pair name to function parameters

Reviewer: ```suggestion
- Access all functions in `AggregatorProxy` by including price pair name as parameter
```

```solidity
// initialize

> import { IAggregatorRouter } from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregatorRouter.sol";

Reviewer: I think it would be good to include code that can be copy-pasted and run without modifications, or split it to multiple code blocks. The second options is probably better so we don't have to come up with some scenario when it should be used. We can do the following.

Initialize `AggregatorRouter` that enables access to all supported data feeds.

```Solidity
import { IAggregatorRouter } from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregatorRouter.sol";
contract DataFeedConsumer {
    IAggregatorRouter internal router;
    constructor(address _router) {
        router = IAggregatorRouter(_router);
    }
}
```

Read the latest submitted value of given data feed (e.g. `"BTC-USDT"`)

```Solidity
(
   uint80 id,
    int256 answer,
    uint startedAt,
    uint updatedAt,
    uint80 answeredInRound
) = router.latestRoundData("BTC-USDT");
```

Read value submitted to a given data feed (e.g. `"BTC-USDT"`) for specific `roundId`.

```Solidity
uint80 roundId =
(
    uint80 id,
    int256 answer,
    uint startedAt,
    uint updatedAt,
    uint80 answeredInRound
) = router.getRoundData("BTC-USDT", roundId);
```

Get decimals for given data feed (e.g. `"BTC-USDT"`)

```Solidity
uint8 decimals = router.decimals("BTC-USDT");
```

Get `Aggregator` address associated with given data feed (e.g. `"BTC-USDT"`).

```Solidity
address currentAggregator = router.aggregator("BTC-USDT")
```
